learning/tutorial-five.php

```diff
@@ -32,14 +32,12 @@
 				$('#result').html(data);
 			});
 			*/
-			/*
 			//gets JSON encoded data 
 			$.getJSON('users.json', function(data){
 				$.each(data, function(i, user){
-					$('ul#users').append('<li>'+user.firstName+'</li>');
+					$('ul#users').append('<li class="'+user.lastName+'">'+user.firstName+'</li>');
```

Comment:
this code is not part of the status bar.

Reply:
okay done